### ISSUE TITLE
`DefaultPlugins` documentation is out of date

### ISSUE THREAD

**Issue by matiqo15** [bevyengine/bevy#11453]
## How can Bevy's documentation be improved?

Documentation of [default plugins](https://github.com/bevyengine/bevy/blob/main/crates/bevy_internal/src/default_plugins.rs#L3-L37) is missing `AccessibilityPlugin`.

Also see #7332
